### ISSUE TITLE
after_success and before_deploy needs to be disabled in moban job

### ISSUE THREAD

**Issue by blazeu** [coala/git-task-list#54]
It's causing the job to fail https://travis-ci.org/coala/git-task-list/jobs/390636589

**Comment by jayvdb** [comment]
Still happening ... maybe `deploy` is also a problem...

**Comment by jayvdb** [comment]
yup , confirmed on master branch on my fork .. https://travis-ci.org/jayvdb/gsoc-prep-tasks/jobs/390806894
